packages/did-core-test-server/package.json

```diff
@@ -10,7 +10,7 @@
     "serve": "node run.js",
     "generate-test-data": "node report/generate-test-data.js",
     "generate-report": "node report/generate-report.js",
-    "test-and-generate-report": "node report/generate-test-data.js; node report/generate-report.js",
+    "test-and-generate-report": "node report/generate-test-data.js; node report/generate-report.js; mv ./report/index.html ../../docs",
```

Comment:
Won't this fail because the /docs directory doesn't exist? Do we need an `mkdir -p` in here?

Reply:
Oh, wait, nm... I think I missed: https://github.com/w3c/did-test-suite/pull/158/files#diff-d0d77eb77bc0a80bc42c75176cbc8251eb4795b7e28a0bdd3a282a612856d4daR25

Reply:
I added a README to it, explaining its purpose, you can stuff other stuff in that directory, like images, or other files if needed.